core/input/input.cpp

```diff
@@ -187,7 +187,7 @@ void Input::get_argument_options(const StringName &p_function, int p_idx, List<S
 void Input::VelocityTrack::update(const Vector2 &p_delta_p) {
 	uint64_t tick = OS::get_singleton()->get_ticks_usec();
 	uint32_t tdiff = tick - last_tick;
-	float delta_t = tdiff / 1000000.0;
+	float delta_t = tdiff / 1'000'000.0;
```

Comment:
For floating-point literals, it's worth considering the exponential form `1e6` instead.